src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const total = days * 40;
+
+  if (days < 3) {
+    return total;
+  }
+
+  if (days >= 3 && days < 7) {
+    return total - 20;
+  } else {
+    return total - 50;
+  }
```

Comment:
```suggestion
  }
  
  return total - 50;
```
Если ни один из предыдущих if не сработал, значит days >= 7